client/src/components/landing_page/ScorecardSearch.vue

```diff
@@ -6,7 +6,12 @@
       :acceptedTypes='acceptedTypes'
       @result='onGeocodeResults'
     />
-    <button class='gold-button' @click='onSearch' :disabled='!searchHasResult'>
+    <button
+      class='gold-button'
+      id='scorecard-search-button'
```

Comment:
Is there a way to search this button by text rather than adding an id?

Reply:
There probably is. But I didn't see an obvious way to select the button element with text. I tried using a `waitForFunction` like I use for finding text on the page, but the canary failed with:

> TypeError: searchButton.click is not a function

So I went with ID to get it working.